third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -41,6 +41,14 @@ func resourceDataflowFlexTemplateJob() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"zone": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
We probably want `Computed: true` as well, or else we'll see a diff when the provider zone is used.